ramanujan/cmf/known_cmfs/known_cmfs.py

```diff
@@ -112,3 +121,47 @@ def hypergeometric_derived_3d():
             ),
         }
     )
+
+
+def hypergeometric_dervied_3F2():
```

Comment:
are there two CMFs called hypergeometric_derived_3F2?
i think we had 3 CMFs from hypergeos:
1. the Pi one, with offsets 1/2 and z=1/2 hard coded in it (hypergeometric_derived_2F1 i think it was)
2. the general 2F1 one, with rational functions, and z as free param (hypergeometric_2F1_3d i called it.) - might still be in my branch i have yet to merge
3. the new 3F2 one with 3x3 mats.  (hypergeometric_dervied_3F2)
then what is this one?

Reply:
Good catch, I removed the dup one, and also fixed a type (dervied -> derived).
I think in the future we might wanna change the 2F1 one to the canonical form instead of the a, b, c form.